packages/web/resolvers/thanks.ts

```diff
@@ -85,6 +91,93 @@ const ThanksMutations = extendType({
           }
         })
 
+        const thanksCountQuery = Prisma.sql`
+          SELECT COUNT(*) AS count
+          FROM "CommentThanks" AS ct
+          JOIN "Comment" AS c
+            ON ct."commentId" = c.id
+          WHERE c."authorId" = ${comment.author.id}
+        `
+        
+        const thanksGivenCountQuery = Prisma.sql`
+          SELECT COUNT(*) AS count
+          FROM "CommentThanks" AS ct
+          WHERE ct."authorId" = ${userId}
+        `
+        const newBadgeCount = await assignCountBadges(
+          ctx.db,
+          comment.author.id,
+          thanksCountQuery,
+          {
+            10: BadgeType.TEN_THANKS,
+            50: BadgeType.FIFTY_THANKS,
+            100: BadgeType.ONEHUNDRED_THANKS,
+            250: BadgeType.TWOHUNDREDFIFTY_THANKS,
+            500: BadgeType.FIVEHUNDRED_THANKS,
+            1000: BadgeType.ONETHOUSAND_THANKS,
+            1250: BadgeType.ONETHOUSANDTWOHUNDREDFIFTY_THANKS,
+            1500: BadgeType.ONETHOUSANDFIVEHUNDRED_THANKS,
+            2250: BadgeType.TWOTHOUSANDTWOHUNDREDFIFTY_THANKS,
+            2500: BadgeType.TWOTHOUSANDFIVEHUNDRED_THANKS,
+            5000: BadgeType.FIVETHOUSAND_THANKS,
+            10000: BadgeType.TENTHOUSAND_THANKS,
+          }
+        )
+
+        const newThanksGivenBadgeCount = await assignCountBadges(
+          ctx.db,
+          comment.author.id,
```

Comment:
think this is the bug mentioned on slack, this needs to be `userId` as that's the party that will get the badge assigned to them (even though we're querying against another user's actions).

Reply:
Ah yes! Thanks mate!